### PR TITLE
Merge 8.1.x into develop

### PR DESCRIPTION
**Pull request by matthieu-rolland**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge 8.1.x into develop
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | 
| Fixed ticket?     | 
| Related PRs       | 
| Sponsor company   | PrestaShop SA

### CODE REVIEW

File: src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApplicationController.php [hunk 1]
```diff
@@ -67,70 +59,28 @@ public function indexAction(AuthorizedApplicationsFilters $filters): Response
         ]);
     }
 
-    /**
-     * @param Request $request
-     *
-     * @return Response
-     */
-    public function createAction(Request $request): Response
+    public function viewAction(): void
```
Comment (jolelievre) [inline src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApplicationController.php:71]: @mflasquin seems like it's removing some implemented actions I'm not sure about this I guess the actions were finished and merged in develop after the 8.1.x branch was created, but they shouldn't be removed right?

File: src/PrestaShopBundle/Resources/config/services/bundle/repository.yml [hunk 2]
```diff
@@ -148,3 +138,8 @@ services:
     factory: [ '@doctrine.orm.default_entity_manager', getRepository ]
     arguments:
       - PrestaShopBundle\Entity\AuthorizedApplication
+    public: true
+
+  prestashop.core.admin.feature_flag.repository:
```
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/config/services/bundle/repository.yml]: The alias has been moved e should keep it under the aliased service

**Review by jolelievre — CHANGES_REQUESTED** [review]
Hi @matthieu-rolland and @mflasquin 

There are some changes on the Application controller that seem weird, I suspect a conflict was solved by removing a new feature. Maybe this needs to be fixed

File: src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApplicationController.php (hunk 1 above)
Comment (matthieu-rolland) [reply src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApplicationController.php:71]: that's what it looks like, I should probably put it back in develop, wdyt @mflasquin  ?

Comment (matthieu-rolland) [reply src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/AuthorizationServer/ApplicationController.php:71]: fixed

**Review by matks — COMMENTED** [review]
What a big PR!

I checked the content and it seems globally OK however CI fails

**Review by matks — APPROVED** [review]

File: src/PrestaShopBundle/Resources/config/services/bundle/repository.yml (hunk 2 above)
Comment (matthieu-rolland) [reply src/PrestaShopBundle/Resources/config/services/bundle/repository.yml]: fixed

**Review by mflasquin — APPROVED** [review]

**Review by jolelievre — APPROVED** [review]
Thanks @matthieu-rolland

**Comment by matthieu-rolland** [comment]
Thank you everyone:

https://github.com/matthieu-rolland/ga.tests.ui.pr/actions/runs/4132135466

**Comment by nicosomb** [comment]
404, I think that the correct URL is https://github.com/matthieu-rolland/ga.tests.ui.pr/actions/runs/4132204535

**Comment by matthieu-rolland** [comment]
test suite OK

to be reviewed by @PrestaShop/ui-tests-maintainers :+1:

**Review by boubkerbribri — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you everyone :+1:

**Pull request merged**